### PR TITLE
Added validation for resource id in chainbridge frame

### PR DESCRIPTION
**Pull request by karthikeyan-cere**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - Added dynamic block interval
+- Added validation for resource Id in chainbridge frame.
```
Comment (AndreiNavoichyk) [inline CHANGELOG.md:13]: Increase spec_version.

File: frame/chainbridge/src/tests.rs [hunk 2]
```diff
@@ -183,6 +185,34 @@ fn asset_transfer_success() {
...
 }
 
+#[test]
+fn asset_transfer_invalid_resource_id() {
+    new_test_ext().execute_with(|| {
+        let dest_id = 2;
+        let to = vec![2];
+        let resource_id = [1; 32];
+        let amount = 100;
+        assert_ok!(Bridge::set_threshold(Origin::root(), TEST_THRESHOLD,));
```
Comment (AndreiNavoichyk) [inline frame/chainbridge/src/tests.rs]: Add a newline between the arrange and act/assert sections.

Comment (AndreiNavoichyk) [reply frame/chainbridge/src/tests.rs]: Group bridge set up (set_threshold, whitelist_chain) into 1 section without newlines.

**Review by AndreiNavoichyk — APPROVED** [review]

File: frame/chainbridge/src/lib.rs [hunk 3]
```diff
@@ -551,6 +551,10 @@ impl<T: Trait> Module<T> {
             Self::chain_whitelisted(dest_id),
             Error::<T>::ChainNotWhitelisted
         );
+        ensure!(
```
Comment (AndreiNavoichyk) [inline frame/chainbridge/src/lib.rs:554]: T

File: CHANGELOG.md (hunk 1 above)
Comment (karthikeyan-cere) [reply CHANGELOG.md:13]: increased by 1

File: frame/chainbridge/src/tests.rs (hunk 2 above)
Comment (karthikeyan-cere) [reply frame/chainbridge/src/tests.rs]: Updated the code.

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**